includes/payments/functions.php

```diff
@@ -840,7 +840,10 @@ function edd_get_total_earnings() {
 
 				if( ! empty( $payments ) ) {
 					$payments = implode( ',', $payments );
-					$total += $wpdb->get_var( "SELECT SUM(meta_value) FROM $wpdb->postmeta WHERE meta_key = '_edd_payment_total' AND post_id IN({$payments})" );
+
+					foreach( $payments as $payment_id ) {
```

Comment:
I wish I could, but there's a chance that the totals calculated so far have always been wrong (at least, in EDD 2.5). If you have an order for 100 USD and one for 100 EUR, the logic calculates the total earnings as 100 + 100, which is incorrect. Having the possibility to recalculate them from scratch would allow to rectify this, to a certain extent.